typescript/kurtosis_core_rpc_api_bindings/api_container_service_pb.d.ts

```diff
@@ -629,6 +629,9 @@ export class WaitForEndpointAvailabilityArgs extends jspb.Message {
   getServiceId(): string;
   setServiceId(value: string): void;
 
+  getHttpMethod(): WaitForEndpointAvailabilityArgs.HttpMethodMap[keyof WaitForEndpointAvailabilityArgs.HttpMethodMap];
+  setHttpMethod(value: WaitForEndpointAvailabilityArgs.HttpMethodMap[keyof WaitForEndpointAvailabilityArgs.HttpMethodMap]): void;
```

Comment:
@leoporoli Typescript has its own equivalent to the constructor objects, and now that Typescript is merged we need to maintain it too. So you'll need to update the `newEndpointAvailabilityArgs` "constructor" in Typescript to also use the httpMethod

Reply:
ok